README.md

```diff
@@ -325,32 +325,24 @@ var config = AirbrakeConfig.Load(settings);
 
 ### AirbrakeNotifier
 
-#### Notify
+#### CreateNotice
```

Comment:
What do you think about renaming this to `BuildNotice`? We have this method in some of our notifiers, so it should be familiar to old users.

Reply:
Good idea. Done.